app/webhook/webhook.go

```diff
@@ -70,22 +72,22 @@ func (w *Webhook) Post(entry ReportListEntry) {
 		return
 	}
 
-	listUrl := fmt.Sprintf("https://crash.pmmp.io/list?min=%d&max=%d", w.reportMinId, w.reportMaxId)
+	listUrl := fmt.Sprintf("%s/list?min=%d&max=%d", w.baseURL, w.reportMinId, w.reportMaxId)
 
 	messageText := make([]string, 0, 20)
 	for _, entry := range w.reportList {
-		messageText = append(messageText, fmt.Sprintf("<https://crash.pmmp.io/view/%d|#%d: %s>", entry.ReportId, entry.ReportId, entry.Message))
+		messageText = append(messageText, fmt.Sprintf("<%s/view/%d|#%d: %s>", w.baseURL, entry.ReportId, entry.ReportId, entry.Message))
 	}
 	t := strings.Join(messageText, "\n")
 	if w.reportCount > uint32(cap(w.reportList)) {
-		t += fmt.Sprintf("\n\n%d more reports not shown. <%s|View the full list>", w.reportCount - uint32(cap(w.reportList)), listUrl)
+		t += fmt.Sprintf("\n\n%d more reports not shown. <%s|View the full list>", w.reportCount-uint32(cap(w.reportList)), listUrl)
```

Comment:
Is there any specification on the spacing style here?

Reply:
Personally I've been using spaces for as long as I can remember. The cramped style like this hurts readability in my opinion.

Reply:
GoLand removes spaces automatically sometimes, but I don't understand when exactly.

Reply:
`go fmt` removes spaces here iirc

Reply:
Oh boy, opinionated default styling. Is there a way to change that?

Reply:
@dktapps One of the philosophies is "Stop arguing over which style to use; use our default style guide!" gofmt is even more authoritarian than me!